### DISCUSSION TITLE
General Discussion : Free Live Online JavaScript Classes

### DISCUSSION THREAD

**Discussion by Jay-study-nildana** [Jay-study-nildana/FrontEndForStudents#11] [Free Live Online JavaScript Classes]
Dear students,

Please use this thread to ask any questions about the free online classes.

You can also join the [discord group](https://discord.gg/CVugpJRBTB) and ask questions over chat.

Thanks

Jay.

**Comment by Naik1524** [comment]
I want to join

**Reply by Jay-study-nildana** [reply]
@Naik1524 

Join the discord group. I am waiting for at least 5 students to show interest (and join discord group) and I will setup the batch, schedule, time, dates.

Right now, the plan is to have sessions at 10 PM IST, Tuesday to Thursday. Plus doubts sessions on other days. Feel free to invite your friends, and also add your own ideas and suggestions. 

If you want to help and get involved behind the scenes, send me an email and I will get in touch with you. 

Thanks.

Jay.

**Reply by asmallick1999** [reply]
Interested

**Reply by Jay-study-nildana** [reply]
@asmallick1999 

already saw you on discord, and added to the channel. Thanks for joining.

**Reply by asmallick1999** [reply]
Thank you sir

**Comment by Jay-study-nildana** [comment]
Folks,

This is going away. Next

https://github.com/Jay-study-nildana/FrontEndForStudents/discussions/20

Thanks

Jay
